terraform-plans/configs/dcgm-snap_main.tfvars

```diff
@@ -22,7 +22,9 @@ templates = {
   promote = {
     source      = "./templates/github/snap_promote.yaml.tftpl"
     destination = ".github/workflows/promote.yaml"
-    vars        = {}
+    vars = {
```

Comment:
Maybe it's not scope of this PR, but because we have this situation of the `core24` I would like to see how we are going to approach this situation.

On release.yaml we are releasing just for `latest/edge`, which means that just accessing the store we would be able to get what is from latest/edge and pass to core24/edge

In summary, I think we should have a new template variable that will be used in a matrix for the releases and in this case releasing on both `core24` and `latest/edge`.